### PR TITLE
Add support for multiple handlers

### PR DESCRIPTION
**Pull request by mpivaa**
Why?

Because we need to support the selection of a range of values.

ezgif com-crop
ezgif com-crop 1

I know, it's ugly, so as the code.

### CODE REVIEW

**Comment by futrica** [comment]
:heart_eyes:

File: lib/react_timeline_slider.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        <G className="timeline" ref="timeline" transform="translate(25, 15)">
+          <G className="ticks">
+            {this.renderTicks()}
+          </G>
+          <G className="handlers">
+            <Handler ticks={this.ticks()}
+              defaultValue={this.value(0)}
+              onValueChange={this.valueChanged(0).bind(this)}/>
+            {(() => {
+              if(this.props.multiple) {
```
Comment (jaimerson) [inline lib/react_timeline_slider.jsx]: Não sabia que rolava isso

**Comment by jaimerson** [comment]
:ship:

**Comment by mpivaa** [comment]
Me ajudem a refatorar isso hehe.

**Comment by mpivaa** [comment]
It's not good enough.

**Pull request closed without merging**